.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_GIPHY_API_KEY=OWyGqcD3SWfggMMzdErRe9yHvasf2BBq
```

Comment:
you shouldn't save the api key in the .env.example

Reply:
It's ok, it's just an example, It's a fake key

Reply:
oh, ok haha nice.